src/components/ImageView/ImageView.js

```diff
@@ -746,11 +746,20 @@ export default observer(
     }, 16);
 
     componentDidMount() {
+      const { store, item } = this.props;
+      const annotation = store.annotationStore.selected;
+
       window.addEventListener("resize", this.onResize);
-      this.attachObserver(this.props.item.containerRef);
+      this.attachObserver(item.containerRef);
       this.updateReadyStatus();
 
       hotkeys.addDescription("shift", "Pan image");
+      hotkeys.addNamed("polygon:undo", () => {
+        if (annotation.isDrawing) item.drawingRegion.undoPoints();
```

Comment:
`DrawingRegion` is used not only in polygons, so you can get an error in this line. but I think it would be enough to just add `?.`